fusion_docs/licensing.md

```diff
@@ -24,4 +24,5 @@ A Fusion license sets a specific quota for Fusion use, measured by total pipelin
 
 ### For Seqera Enterprise instances without network connectivity, and direct Fusion use with Nextflow  
 
-[Contact Seqera](https://seqera.io/contact-us) for a license that can be integrated into your workflow, or to set up an evaluation.
+- Run Fusion by using a Cloud API via Nextflow’s `TOWER_ACCESS_TOKEN` environment variable in the `nextflow.config` file.
+- This will count toward the Free/Paid usage associated with your Cloud organization.
```

Comment:
```suggestion
- Throughput counts towards the free or paid usage associated with your Cloud organization.
```
I find starting the bullet point with "This" to be unclear. Suggestion to rephrase to make it clear it's the throughput that is being measured.